build-tests/sparo-outputs/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "sparo-outputs",
```

Comment:
Consider a naming pattern such as `sparo-output-test` to make it clear that this is a test project.